cmd/pulumi-resource-docker-buildkit/main.go

```diff
@@ -264,15 +264,31 @@ func (k *dockerBuildkitProvider) dockerBuild(
 	for _, v := range inputs["platforms"].ArrayValue() {
 		platforms = append(platforms, v.StringValue())
 	}
-	cmd := exec.Command(
-		"docker", "buildx", "build",
+
+	var args []string
+	for _, v := range inputs["args"].ArrayValue() {
+		args = append(args, "--build-arg")
+		args = append(args, fmt.Sprintf("%s=%s", v.ObjectValue()["key"].StringValue(), v.ObjectValue()["value"].StringValue()))
+	}
+
+	var arguments []string = []string{
+		"buildx", "build",
 		"--platform", strings.Join(platforms, ","),
 		"--cache-from", name,
 		"--cache-to", "type=inline",
 		"-f", filepath.Join(context, dockerfile),
 		"--target", target,
 		"-t", name, "--push",
-		context,
+	}
+
+	if len(args) > 0 {
+		arguments = append(arguments, args...)
+	}
```

Comment:
Can you move the for loop here to avoid introducing the `args` variable? Also, no need to check for empty explicitly.